lib/HyloJWT.js

```diff
@@ -0,0 +1,34 @@
+import { omit } from 'lodash/fp'
+import jwt from 'jsonwebtoken'
+import { getPublicKeyFromPem } from './util'
+
+export { TokenExpiredError } from 'jsonwebtoken'
+
+export const generateHyloJWT = (sub, data = {}) => {
+  const privateKey = Buffer.from(process.env.OIDC_KEYS.split(',')[0], 'base64')
+
+  return jwt.sign(
+    {
+      iss: process.env.PROTOCOL + '://' + process.env.DOMAIN,
+      aud: 'https://hylo.com',
+      sub,
+      exp: Math.floor(Date.now() / 1000) + (60 * 60 * 4), // 4 hour expiration
+      ...omit(['iss', 'aud', 'sub', 'exp'], data)
```

Comment:
we may actually want to allow overriding these values, but for now this is fine

Reply:
Agreed. I was simply not prematurely optimizing :) A win to get it all in one place for now...